@xen-orchestra/lite/src/locales/fr.json

```diff
@@ -69,11 +86,19 @@
   "switch-theme": "Changer de thème",
   "system": "Système",
   "tasks": "Tâches",
+  "theme-auto": "Auto",
+  "theme-dark": "Sombre",
+  "theme-light": "Clair",
   "top-#": "Top {n}",
   "total-free": "Total libre",
   "total-used": "Total utilisé",
   "unreachable-hosts": "Hôtes inaccessibles",
   "unreachable-hosts-reload-page": "C'est fait. Rafraîchir la page",
   "version": "Version",
+  "vm": {
+    "description": "Description",
+    "name": "Nom",
+    "power-state": "État d'alimentation"
```

Comment:
They will probably be used elsewhere, it would be better not to "link" them to VM and make them "generic".